src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/data-broker-profiles/manual-remove/page.tsx

```diff
@@ -2,101 +2,18 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-import Link from "next/link";
-import styles from "../dataBrokerProfiles.module.scss";
-import { getL10n } from "../../../../../../../../functions/server/l10n";
-import buttonStyles from "../../../../../../../../components/server/button.module.scss";
+import { getServerSession } from "next-auth";
+import { authOptions } from "../../../../../../../../api/utils/auth";
+import { redirect } from "next/navigation";
+import { getUserBreaches } from "../../../../../../../../functions/server/getUserBreaches";
+import { ManualRemoveView } from "./ManualRemoveView";
 
-export default function ManualRemove() {
-  const l10n = getL10n();
+export default async function ManualRemove() {
+  const session = await getServerSession(authOptions);
+  if (!session?.user?.subscriber?.id) {
+    return redirect("/");
+  }
```

Comment:
Probably very unrelated to this code and PR in general, but so far we have this pattern about 6&times; in this PR alone. Might be worth extracting out into some helper or middleware eventually.

---

Looks like about 12 total usages `!session?.user?.` &mdash; according to this gem:
```sh
git grep -Pn "\!session\??\.user\??\.subscriber"
```